pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -193,199 +221,87 @@ def get_commuter_data(read_data=dd.defaultDict['read_data'],
-                    # the first or is for the case that the right first line of the incoming people directly
-                    # addresses one
-                    # the latter 'or's is used if no single county nor gov region of a federal state is stated
-                    # explicitly
-                    # although there are existent government regions in this federal state (i.e., the state itself is
-                    # not considered a governement region according to gov_list)
-                    elif ((str(commuter_migration_file.iloc[i][3]) == 'Übrige Regierungsbezirke (Bundesland)' and str(
-                            commuter_migration_file.iloc[i][4]).isdigit())
-                          or ((commuter_migration_file.iloc[i][2]).isdigit() and str(
-                              commuter_migration_file.iloc[i - 1][2]).startswith('nan'))
-                          or (len(str(commuter_migration_file.iloc[i][2])) == 2 and
-                              abs(float(commuter_migration_file.iloc[i][2]) - float(
-                                  commuter_migration_file.iloc[i - 1][2])) == 1)
-                          or (len(str(commuter_migration_file.iloc[i][2])) == 2 and
-                              abs(float(commuter_migration_file.iloc[i][2]) - float(
-                                  commuter_migration_file.iloc[i - 1][2])) == 2)):
-
-                        # auxiliary key of Bundesland (key translated to int starting at zero)
-                        dummy_key = int(commuter_migration_file.iloc[i][2]) - 1
-
-                        # sum population of all counties not explicitly migrated
-                        # from the current gov region migrated from
-                        dummy_pop_sum = 0
-                        for j in range(0, len(state_gov_table[dummy_key])):
-                            # over all government regions not explicitly stated
-                            gov_index = govkey2numlist[state_gov_table[dummy_key][j]]
-                            for k in range(0,
-                                           len(gov_county_table[gov_index])):
-                                # over all counties of the considered gov region
-                                if counties_migratedfrom[gov_index][k] < 1:
+    with progress_indicator.Spinner() as p:
+        for i in range(len(commuter_migration_files)):
+            p.set_message('Progress '+str(i+1)+'/16')
+            commuter_migration_file = commuter_migration_files[i]
+
+            counties_done = []  # counties considered as 'migration from'
+            # current_row = -1  # row of matrix that belongs to county migrated from
+            current_col = -1  # column of matrix that belongs to county migrated to
+            checksum = 0  # sum of county migration from, to be checked against sum in document
+
+            for i in range(0, commuter_migration_file.shape[0]):
```

Comment:
U are already using i for a loop in this scope :) 
Also, use more meaningful names to increase comprehensibility.

Additionally, you should be consistent in the notation. According to the former loops, you should use 
for i in range(commuter_migration_file.shape[0]):

Reply:
I replaced a few names. Please have another look.